src/strategyHelpers/canary/canaryHelper.ts

```diff
@@ -222,8 +227,8 @@ async function cleanUpCanary(
             const canaryObjectName = getCanaryResourceName(name)
             const baselineObjectName = getBaselineResourceName(name)
 
-            await deleteObject(kind, canaryObjectName)
-            await deleteObject(kind, baselineObjectName)
+            await deleteObject(kind, canaryObjectName, ns)
```

Comment:
💡 Nit: various places we use `namespaces, for clarity might be `ns` can be renamed to `namespaces`